api_spec.yaml

```diff
@@ -1285,6 +1287,10 @@ paths:
                                                               type: array
```

Comment:
rename this to `providersFromCore` for clarity? Also add a description regarding why we need both.